earthaccess/api.py

```diff
@@ -246,10 +241,8 @@ def get_s3_credentials(
 def collection_query() -> Type[CollectionQuery]:
     """Returns a query builder instance for NASA collections (datasets)
 
-    Parameters:
-        cloud_hosted (Boolean): initializes the query builder for cloud hosted collections.
```

Comment:
Mkdocs was throwing warnings about these parameters which don't exist in the signature. That's awesome, but can we get it to upgrade those to errors so we can protect ourselves from info getting out of sync? There are still some unaddressed warnings after this PR.

Reply:
`mkdocs build --strict` will exit with a return code of 1 and a message with the number of warnings raised.
https://github.com/mkdocs/mkdocs/pull/1613#issuecomment-416703384

Reply:
Nice! I'll open an issue referencing this comment :)